src/librustc_mir/build/expr/stmt.rs

```diff
@@ -240,16 +240,13 @@ impl<'a, 'gcx, 'tcx> Builder<'a, 'gcx, 'tcx> {
                            temp, expr, this.block_context);
                     place
                 };
-                unpack!(block = this.into(&temp, block, expr));
-
-                // Attribute drops of the statement's temps to the
-                // semicolon at the statement's end.
```

Comment:
wow, if I'm reading al this right, even though you got rid of all that `StatementSpan` stuff, you didn't have to change any of the tests that are testing the attribution of drops of a statement's temps to the semicolon at the statements end, right?

That's great.